languages.toml

```diff
@@ -2952,3 +2952,18 @@ file-types = ["log"]
 [[grammar]]
 name = "log"
 source = { git = "https://github.com/Tudyx/tree-sitter-log", rev = "62cfe307e942af3417171243b599cc7deac5eab9" }
+
+[[grammar]]
+name = "hoon"
+source = { git = "https://github.com/urbit-pilled/tree-sitter-hoon", rev = "1d5df35af3e0afe592832a67b9fb3feeeba1f7b6" }
+
+[[language]]
+name = "hoon"
+scope = "source.hoon"
+grammar = "hoon"
```

Comment:
This key can be omitted since the `[[grammar]]` entry uses the same name as the `[[language]]` name